Helper/ArrayHelper.php

```diff
@@ -67,6 +67,8 @@ public static function getValueFromArray($array, $key, $default = null)
     /**
      * Get incrementId and quoteId from display_id.
      *
+     * No longer used, leaving in for now for tests, will remove
```

Comment:
Do we still need this function for unit tests?

Reply:
Good catch, forgot about this one. I'll remove it and see what breaks, I'm pretty sure I've pulled it out of the tests though.